ocfweb/main/home.py

```diff
@@ -12,7 +12,7 @@
 from ocfweb.component.lab_status import get_lab_status
 
 
-@periodic(60, ttl=86400)
+@periodic(60)
```

Comment:
why this change?

this was intentional so that when people screw up the file format (happens all the time) the home page will just have stale date (rather than error out)

the `60` means it still gets updated every 60 seconds when everything is happy